src/components/CommentInfo/CommentInfo.jsx

```diff
@@ -1,3 +1,22 @@
-export const CommentInfo = () => (
-  <>Put the comment here</>
+export const CommentInfo = ({ comment }) => (
```

Comment:
The prop 'comment' is not validated. It's a good practice to validate props using PropTypes in React.